react/components/Slider.js

```diff
@@ -114,40 +112,30 @@ class Slider extends Component {
     this.init()
   }
 
-  enableTransition = (extraStyles = {}) => {
-    const { easing, duration } = this.props
-
-    setStyle(this._sliderFrame.current, {
-      ...getStylingTransition(easing, duration),
-      ...extraStyles
-    })
-  }
-
-  disableTransition = (extraStyles = {}) => {
-    const { easing } = this.props
-    setStyle(this._sliderFrame.current, {
-      ...getStylingTransition(easing),
-      ...extraStyles
-    })
-  }
-
   init = () => {
-    const { draggable, currentSlide, loop, cursor } = this.props
+    const { draggable, currentSlide } = this.props
     this.setSelectorWidth()
     this.setInnerElements()
     this.perPage = resolveSlidesNumber(this.props.perPage)
 
-    this.enableTransition({
-      width: `${(this.selectorWidth / this.perPage) * this.innerElements.length}px`,
-      ...(draggable ? { cursor } : {}),
+    requestAnimationFrame(() => {
```

Comment:
I guess this `requestAnimationFrame` is needed because this function is being called on the constructor, so the elements are not properly initialized until the next frame.

This is not a good practice—it's better to call this on `componentDidMount`. There is no guarantee on when the constructor will be actually called on React components, so in a different React version this function might be called when there are no elements rendered yet.

(I know this is not directly related to this PR, but I think this is a good time to fix this)

Reply:
This funciton is not being called on the constructor, is just called on `componentDidUpdate`, it was called before on `componentDidMount` too, thats why this code is wrapped in a function, but now is just needed on `componentDidUpdate` and doesn't make any sense to exist, I guess I forgot to remove it